src/kubernetes/meta_cache.rs

```diff
@@ -0,0 +1,99 @@
+use std::collections::HashMap;
+
+use kube::core::ObjectMeta;
+
+#[derive(Default)]
+pub struct MetaCache {
+    pub cache: HashMap<MetaDescribe, bool>,
```

Comment:
Maybe HashSet is enough. Just remove meta when receiving Deleted.

Reply:
I need second field (in this case - bool), for mark object for deletion if `Delete` event occurred.
But if after `Delete` event, happening event `Create` for the same object, then bool changed back

Reply:
I think i understood what do you mean

Yeah, you right, i replace HashMap by HashSet